Mathlib/Algebra/Polynomial/FieldDivision.lean

```diff
@@ -524,7 +525,8 @@ theorem coe_normUnit_of_ne_zero [DecidableEq R] (hp : p ≠ 0) :
   have : p.leadingCoeff ≠ 0 := mt leadingCoeff_eq_zero.mp hp
   simp [CommGroupWithZero.coe_normUnit _ this]
 
-theorem normalize_monic [DecidableEq R] (h : Monic p) : normalize p = p := by simp [h]
+theorem normalize_monic [DecidableEq R] (h : Monic p) : normalize p = p :=
+  h.normalize_eq_self
```

Comment:
Note that `normalize_monic` and `Monic.normalize_eq_self` are exactly identical.

Reply:
Should one become an alias of the other? Or even deprecated?

Reply:
I agree there should be at least an `alias` set up, but there's enough bikeshedding in there that I wanted to postpone the specifics to another PR :)

Reply:
I picked `normalize_monic` to become a deprecated alias, mostly arbitrarily. Please bikeshed, if desired, at #18024 :)